### ISSUE TITLE
blutooth configration

### ISSUE THREAD

**Issue by samir176520** [sejjy/mechabar#5]
first thank you for exellent configration for waybar 
i add bluetooth.jsonc module in my configurations file in place off date but there is issue that background of text looks darkest can you recommend tutorial or documentation to understand for solving this issue and thanks again for you effort

**Comment by sejjy** [comment]
Hi Samir,

Thank you for reaching out. Have you tried applying CSS to your `bluetooth` module? If so, it would be helpful if you could send a screenshot of your Waybar and/or paste the contents of your `config.jsonc` and `style.css` files so I can better understand your issue.

Looking forward to your response :)

Best,
Jesse

**Comment by samir176520** [comment]
hi Jesse
thanks for answer and waiting for my response 
in the included files i tried to change module of date to be blutooth module problem is in background color for this module isn't the same as data module was, i know some stuff in css but not good at all so if you can point where is the bug it will help and thanks again

Best wishes,
Samir

[config.jsonc](https://github.com/user-attachments/files/17669298/config.jsonc)

this is css file
`/* Global Styling */
* {
  font-family: "JetBrainsMono Nerd Font";
  font-weight: bold;
  font-size: 10px;
  min-height: 10px;
}

@import "theme.css";

/* Main Background */
window#waybar {
  background: @main-bg;
}

/* Drop Shadow */
window#waybar > box {
  background-color: transparent;
  box-shadow: 0px 0px 2px 1px rgba(0, 0, 0, 1);
  margin: 2px 2px 2px 2px;
}

/* Tooltip Styling */
tooltip {
  color: @main-fg;
  background: @main-bg;
  border: solid;
  border-width: 1.5px;
  border-radius: 7px;
  border-color: @main-br;
}

/* Workspace Button */
#workspaces button {
  color: @module-fg;
  border-radius: 8px;
  box-shadow: none;
  margin-top: 2px;
  margin-right: 0px;
  margin-bottom: 2px;
  margin-left: 0px;
  padding: 0px;
  padding-right: 2px;
  padding-left: 2px;
  animation: ws_normal 20s ease-in-out 1;
}

/* Active Workspace Button */
#workspaces button.active {
  color: @active-fg;
  background: @active-bg;
  text-shadow: 0px 0px 2px rgba(0, 0, 0, 0.6);
  box-shadow: 0px 0px 2px 1px rgba(0, 0, 0, 1);
  margin-right: 2px;
  margin-left: 2px;
  padding-right: 6px;
  padding-left: 6px;
  animation: ws_active 20s ease-in-out 1;
  transition: all 0.4s cubic-bezier(0.55, -0.68, 0.48, 1.682);
}

/* Hover Effect for Workspace Buttons */
#workspaces button:hover {
  color: @hover-fg;
  background: @hover-bg;
  box-shadow: none;
  animation: ws_hover 20s ease-in-out 1;
  transition: all 0.3s cubic-bezier(0.55, -0.68, 0.48, 1.682);
}

/* General Module Styling */
#custom-ws,
#workspaces,
#window,
#custom-cpuinfo,
#memory,
#custom-cpu,
#clock,
#custom-wifi,
#custom-media,
#pulseaudio,
#custom-backlight,
#battery,
#bluetooth,
#custom-power {
  opacity: 1;
  color: @module-fg;
  margin-bottom: 0px;
  padding-right: 4px;
  padding-left: 4px;
  text-shadow: 0px 0px 2px rgba(0, 0, 0, 1);
}

/* General Left Arrow Styling */
#custom-left1,
#custom-left2,
#custom-left3,
#custom-left4,
#custom-left5,
#custom-left6,
#custom-left7,
#custom-left8 {
  font-size: 11pt;
  text-shadow: -1px 0px 2px rgba(0, 0, 0, 1);
}

/* General Right Arrow Styling */
#custom-right1,
#custom-right2,
#custom-right3,
#custom-right4,
#custom-right5 {
  font-size: 11pt;
  padding-right: 3px;
  text-shadow: 1px 0px 2px rgba(0, 0, 0, 1);
}

/* Window Icon */
#custom-ws {
  background: @main-bg;
}

/* Workspaces */
#custom-left1 {
  color: @workspaces;
  background: @main-bg;
  padding-left: 2px;
}

#workspaces {
  background: @workspaces;
}

#custom-right1 {
  color: @workspaces;
  background: @main-bg;
}

/* Temperature */
#custom-paddc {
  padding-right: 22px;
}

#custom-left2 {
  color: @cpuinfo;
  background: @main-bg;
  border-radius: 10px;
  padding-left: 3px;
}

#custom-cpuinfo {
  background: @cpuinfo;
  padding-left: 1px;
  padding-right: 0px;
}

/* Memory */
#custom-left3 {
  color: @memory;
  background: @cpuinfo;
  padding-left: 3px;
}

#memory {
  background: @memory;
  padding-left: 1px;
  padding-right: 0px;
}

#memory.warning {
  color: @warning;
}

#memory.critical {
  color: @critical;
}

/* CPU */
#custom-left4 {
  color: @cpu;
  background: @memory;
  padding-left: 3px;
}

#custom-cpu {
  background: @cpu;
}

#custom-leftin1 {
  color: @cpu;
  font-size: 12.2pt;
  margin-bottom: -2px;
}

/* Arch Logo */
#custom-left5 {
  color: @arch-bg;
  background: @main-bg;
  text-shadow: -1px 0px 2px rgba(0, 0, 0, 0.6);
  padding-left: 3px;
}

#idle_inhibitor {
  color: @arch-fg;
  background: @arch-bg;
  font-size: 11pt;
  margin-right: -1px;
  margin-bottom: -2px;
  padding-right: 0px;
  padding-left: 3px;
  text-shadow: 0px 0px 1.5px rgba(0, 0, 0, 1);
}

#custom-right2 {
  color: @arch-bg;
  background: @main-bg;
}

/* Time */
#custom-rightin1 {
  color: @time;
  font-size: 12.2pt;
  margin-bottom: -2px;
}

#clock.time {
  background: @time;
}

#custom-right3 {
  color: @time;
  background: @date;
}

/* Date */
#custom-right4 {
  background: @memory;
}

#custom-bluetooth {
  color: @date;
  background: @memory;
}

#custom-wifi {
  background: @wifi;
  padding-right: 6px;
  padding-left: 4px;
}

#custom-right5 {
  color: @wifi;
  background: @main-bg;
}

/* Media Info */
#custom-media {
  background-color: @main-bg;
  padding-right: 8px;
  padding-left: 8px;
}

/* PulseAudio */
#custom-left6 {
  color: @pulseaudio;
  background: @main-bg;
  padding-left: 3px;
}

#pulseaudio {
  background: @pulseaudio;
}

/* Brightness */
#custom-left7 {
  color: @backlight;
  background: @pulseaudio;
  padding-left: 2px;
}

#custom-backlight {
  background: @backlight;
}

/* Battery */
#custom-left8 {
  color: @battery;
  background: @backlight;
  padding-left: 2px;
}

#battery {
  background: @battery;
}

#battery.full {
  color: @full;
}

#battery.good {
  color: @module-fg;
}

#battery.warning {
  color: @warning;
}

#battery.critical {
  color: @critical;
}

#battery.charging {
  color: @charging;
}

/* Power Button */
#custom-leftin2 {
  color: @battery;
  background: @main-bg;
  font-size: 12.2pt;
  margin-bottom: -2px;
}

#custom-power {
  color: @main-bg;
  background: @power;
  text-shadow: 0px 0px 2px rgba(0, 0, 0, 0.6);
  box-shadow: 0px 0px 2px 1px rgba(0, 0, 0, 1);
  border-radius: 20px;
  margin-top: 2px;
  margin-right: 4px;
  margin-bottom: 2px;
  padding-right: 6px;
  padding-left: 9px;
}

#custom-power:hover {
  color: @hover-fg;
  background: @hover-bg;
  box-shadow: none;
  transition: all 0.3s cubic-bezier(0.55, -0.68, 0.48, 1.682);
}`

**Comment by sejjy** [comment]
Hi Samir,

Thank you for your reply. I noticed that the issue was with the `background: @Date;` declaration. It should be `background: @date;` with a lowercase "d."

I also made some changes to the `bluetooth` and `clock` modules for debugging purposes. Feel free to keep these changes if you find them useful.

Here’s the updated configuration:

[bluetooth-configuration.zip](https://github.com/user-attachments/files/17681894/bluetooth-configuration.zip)

If you encounter any more issues or you have questions, don’t hesitate to reach out :)

Best wishes,
Jesse
